### PR TITLE
icp changes

### PR DESCRIPTION
**Pull request by barakda**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: ibm-block-csi-operator.v1.4.0
   namespace: placeholder
   annotations:
-    capabilities: "Basic Install"
+    capabilities: "Seamless Upgrades"
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:7]: that is [not true](https://github.com/IBM/ibm-block-csi-operator/pull/61)

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml [hunk 2]
```diff
@@ -1375,7 +1382,8 @@ spec:
           required:
           - controller
           - node
-          type: object
+          - license
```
Comment (oriyarde) [inline deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: not sure what this is, but any changes in crd should also be done in the generated yaml, and in the latest CSV yamls

Comment (barakda) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: can you specify the relevant files?

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:7]: will be lint override on internal files

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: https://github.com/IBM/ibm-block-csi-operator/search?q=ibmblockcsis.csi.ibm.com
(excluding paths with a previous version number)

Comment (barakda) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: updated

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:7]: updated - back to original value

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: **excluding paths with a previous version number**.
as I [mentioned before](https://github.com/IBM/ibm-block-csi-operator/pull/106#discussion_r507141902), we are not allowed to change CSV yamls of previous versions.

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:7]: still doesn't look like the original value

File: README.md [hunk 3]
```diff
@@ -150,6 +150,95 @@ multipath -ll
 
 3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
+<br/>
+<br/>
+<br/>
+
+# SecurityContextConstraints RequirementsThe operator uses the restricted and privileged SCC for deployments. ### Custom SecurityContextConstraints definition:<details>
```
Comment (oriyarde) [inline README.md:157]: this doesn't look right. please compare it to the source

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml]: @oriyarde , pls check

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (barakda) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: @oriyarde , pls check

File: README.md (hunk 3 above)
Comment (barakda) [reply README.md]: @oriyarde , pls check

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (oriyarde) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: @barakda please go over the diff and make sure there are no changes to [CSV yamls of previous versions](https://github.com/IBM/ibm-block-csi-operator/pull/112/files#diff-391ed340aea63b1b88ac863444be8c9b186ef4dcf17901083fb50c7b9486730c)

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml (hunk 2 above)
Comment (barakda) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: @oriyarde , have you found another place that the license needs to be added ?

Comment (oriyarde) [reply deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml:1385]: no, just extra change in non-1.4.0

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**